Docs/sphinx_documentation/source/AMReX_Profiling_Tools.rst

```diff
@@ -157,20 +164,24 @@ time an entire function. For example:
     }    // <------ Timer goes out of scope here, calling stop and returning the function time.
 
 Note that all AMReX timers are scoped and will call "stop" when the corresponding object is destroyed.
-This macro is unique because it can _only_ stop when it goes out of scope.
+This macro is unique because it can *only* stop when it goes out of scope.
+
+.. HACK check meaning of _only_ here.
```

Comment:
Were the underscores on "only" for emphasis?

Reply:
Yup. So, I agree with that change.

Reply:
Then I think we should delete this line.

Reply:
got it.